### ISSUE TITLE
check_icmp throws an error on "Address unreachable" [sf#3518671]

### ISSUE THREAD

**Issue by monitoring-user** [monitoring-plugins/monitoring-plugins#1056]
Submitted by bschmidt on 2012-04-17 08:06:29

$ ,/check_ping -H isatap-g.srv.lrz.de -w '200,10%' -c '400,50%' -6 -p 5 --verbose --verbose --verbose --verbose
CMD: /bin/ping6 -n -U -w 10 -c 5 isatap-g.srv.lrz.de
Output: PING isatap-g.srv.lrz.de(2001:4ca0:0:107::a9c:ca0b) 56 data bytes
Output: 
Output: --- isatap-g.srv.lrz.de ping statistics ---
Output: 10 packets transmitted, 0 received, 100% packet loss, time 9057ms
Output: 
PING CRITICAL - Packet loss = 100%|rta=400.000000ms;200.000000;400.000000;0.000000 pl=100%;10;50;0
200.000000:10% 400.000000:50%

$ ./check_ping -H isatap-g.srv.lrz.de -w '200,10%' -c '400,50%' -6 -p 5 --verbose --verbose --verbose --verbose
CMD: /bin/ping6 -n -U -w 10 -c 5 isatap-g.srv.lrz.de
Output: PING isatap-g.srv.lrz.de(2001:4ca0:0:107::a9c:ca0b) 56 data bytes
Output: From 2001:4ca0:0:55::5 icmp_seq=1 Destination unreachable: Address unreachable
Output: 
Output: --- isatap-g.srv.lrz.de ping statistics ---
Output: 4 packets transmitted, 0 received, +1 errors, 100% packet loss, time 3000ms
Output: 
/bin/ping6 -n -U -w 10 -c 5 isatap-g.srv.lrz.de
CRITICAL - Could not interpret output from ping command
